src/error.cpp

```diff
@@ -13,12 +13,21 @@ const char* TacoException::what() const noexcept {
   return message.c_str();
 }
 
-
-ErrorReport::ErrorReport(const char *file, const char *func, int line,
-                         bool condition, const char *conditionString,
-                         Kind kind, bool warning)
-    : msg(NULL), file(file), func(func), line(line), condition(condition),
-      conditionString(conditionString), kind(kind), warning(warning) {
+ErrorReport::ErrorReport(const char* file,
+                         const char* func,
+                         int line,
+                         bool condition,
+                         const char* conditionString,
+                         Kind kind,
+                         bool warning)
+    : msg(NULL)
+    , file(file)
+    , func(func)
+    , line(line)
+    , condition(condition)
+    , conditionString(conditionString)
+    , kind(kind)
+    , warning(warning) {
```

Comment:
Can you undo any change related to clang-format in this file?

Reply:
Yes!